schemas/com.snowplowanalytics.snowplow/media_player_event_playback_rate_change/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event sent when the playback rate has changed.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_playback_rate_change",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
```

Comment:
Worth including `previous_rate` and `new_rate`.

Reply:
✅ added `previousRate` and `newRate`